source/people/main.rst

```diff
@@ -31,13 +32,13 @@ Graduate Students
 |                                    |                                    |                                  |                                |                                       |
 | :doc:`Chelsea D'Angelo <dangelo>`  | :doc:`Patrick Shriwise <shriwise>` | :doc:`Lucas Jacobson <jacobson>` |  :doc:`Moataz Harb <mharb>`    | :doc:`Arrielle Opotowsky <opotowsky>` |
```

Comment:
Can you change the part with Moataz's name to only have 1 space before the `:doc:`? His name is getting formatted differently than everyone else's because it has 2 spaces there.